recharges/models.py

```diff
@@ -4,14 +4,16 @@
 from django.db import models
 
 
-class DummyModel(models.Model):
+class RechargeModel(models.Model):
 
     """
     - Nice docstring please
     """
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    product_code = models.CharField(max_length=20)
-    data = HStoreField(null=True, blank=True)
+    amount = models.DecimalField(max_digits=5, decima_places=2)
+    """data = HStoreField(null=True, blank=True)"""
```

Comment:
It looks like you're trying to comment out this field. The correct way to do that is using the `#` character.  The """ is sometimes used for multiline comments, but mostly for docstrings.  However let's discuss whether we should have this field in the first place tomorrow with Mike.